.github/workflows/tf-destroy.yaml

```diff
@@ -63,7 +63,7 @@ jobs:
         run: |
           terraform init -lock-timeout=300s
           terraform workspace select develop
-          echo "## Terraform Initialised" >> $GITHUB_STEP_SUMMARY
+          echo "## Terraform Initialized" >> $GITHUB_STEP_SUMMARY
```

Comment:
American spelling?

Reply:
Yeah, it reads right to me - initialised means "to use initials" in my head - like "Pete Mallam initialised is PM".

You want it changing?

Reply:
Initialised is valid UK spelling, just wondered on the reason for change :) happy either way.